build_runner/test/entrypoint/run_test.dart

```diff
@@ -146,7 +145,7 @@ main() {
     expect(await runSingleBuild(command, args), ExitCode.success.code);
     expectOutput('web/main.dart.js', exists: false);
     expectOutput('test/hello_test.dart.js', exists: true);
-  });
+  }, solo: true);
```

Comment:
note this probably snuck in by accident

Reply:
Ahh missed that one. Good thing we have lints.